common/app/controllers/IndexControllerCommon.scala

```diff
@@ -6,6 +6,7 @@ import play.api.mvc._
 import services.Index
 import services.IndexPage
 import performance.MemcachedAction
+import views.support.RenderOtherStatus
```

Comment:
out of interest, why `Other`? why not just `RenderStatus`, or if they're all `4xx`s, `RenderErrorStatus`